Tensile/Common.py

```diff
@@ -1434,6 +1468,9 @@ def GetAsmCaps(isaVersion):
   else:
     rv["MaxVmcnt"] = 0
 
+  # TODO- Need to query the max cap, just like vmcnt as well?
+  rv["MaxLgkmcnt"] = 15
+
```

Comment:
Good question - @sdquiring do you know of any limitations for MaxLgkmcnt ? 

Also what is the reasoning behind MaxVmcnt checks at 63 and 15? We could maybe add a little note for each of those numbers.